### DISCUSSION TITLE
problems with 8bitdo usb and ds4

### DISCUSSION THREAD

**Discussion by ghost** [cathery/sys-con#59] [Troubleshooting]
Hello!  I am having problems with 8bitdo usb adapter with ds4 connected.  With the sys-con plugin active, the control loses the rumble function and also the touch button does not work to capture screen.  I looked at the log and apparently it is recognizing the 8bitdo adapter as a ds3 control.  Another detail is that the ds4 led with 8bitdo usb lights up blue.  and with active sys-con it turns green.

**Comment by ghost** [comment]
I downloaded the sys-con plugin to version 0.42 again, and apparently it's working now.  Sometimes 8bitdo works fine, and sometimes not.  when the control led turns green when it turns on, these problems appear.  Normally the led will turn blue when 8bitdo usb is connected.  My adapter is already updated in the latest version, 1.28.  I am using atmosphere and cfw 8.1.

**Comment by cathery** [comment]
That's very strange, considering the sysmodule doesn't detect DS4 controller in any way whatsoever right now. But like others have reported it's a real issue and I don't know what's causing it. Can you send me the log located at `sdmc:/config/sys-con/log.txt`?

**Comment by ghost** [comment]
[log.txt](https://github.com/cathery/sys-con/files/3835923/log.txt)

I added log.txt

I tried using the Xbox 360 Wireless adapter in the tests, along with the ds4 and 8bitdo from usb.

**Comment by cathery** [comment]
I see a few things here:
- Xbox 360 Wireless adapter only ever got used once, yesterday
- Xbox 360 controller got used several times
- Some unrecognized controller got plugged in, but nothing was done to it

Do you know which one is your 8bitdo with ds4?

**Comment by ghost** [comment]
![images (32)](https://user-images.githubusercontent.com/57649848/68676051-e9148e00-0537-11ea-9a05-250ffa9769db.jpeg)

I use this picture. 

In the log there was an event about ds3.  what would it be?  I only tested with the xbox360 wireless adapter, using it with the wireless controller.  I don't have xbox360 wired control. already in the 8bitdo usb adapter I just used it with the ds4.

**Comment by cathery** [comment]
If it noticed a ds3 controller, it would write "Dualshock 3 event went off", but there's no such line in your log. Based on what you said, I'm guessing 8bitdo fakes your ds4 controller as a xbox 360 controller (it turns your light green) and passes it along to my sysmodule, where the rumble doesn't work yet. Please look into the 8bitdo adapter manual and see if you can set it up to recognize your ds4 as a pro controller.

**Comment by ghost** [comment]
Thanks for the reply. I will check then here in the manual if there is something about recognizing as pro controller. Thank you so much for helping me with this issues. =D

**Comment by ghost** [comment]
I checked the manual but found nothing that could help.  I cleared the log and started the console without using ds4, only with the 8bitdo adapter plugged into the dock.  and in the log already appears the adapter with an xbox360 control being connected.  then it recognizes the 8bitdo adapter as xbox360 control.  :(

**Comment by rashevskyv** [comment]
8bitdo used PC mode when sys-con is enabled, so adapter goes to xinput mode. It would be great to have ability to add adapter to some sort of blacklist for module.
